webpack/assets/javascripts/bundle.js

```diff
@@ -42,4 +42,5 @@ window.tfm = Object.assign(window.tfm || {}, {
   advancedFields: require('./foreman_advanced_fields'),
   i18n,
   breadcrumbs: require('./foreman_breadcrumbs'),
+  diffModal: require('./foreman_modal_diff'),
```

Comment:
should this be in the context of config reports? or rather you see this component for everywhere you would like to show a diff in a modal?